src/System.Net.Security/src/System/Net/Windows/CertModule.cs

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Can I assume that any code moved into the /Windows/ folder is exactly what the original code was, has only been moved and not changed at all, etc.?